### PR TITLE
Examples Tonemapping : add background and environment options

### PR DESCRIPTION
**Pull request by vis-prime**
**Description**

Update the tonemapping example with new background and environment intensity/rotation options

Before | After

https://github.com/mrdoob/three.js/assets/119810373/f5555ec1-f490-40cf-b991-14d632eff912

### CODE REVIEW

File: examples/webgl_tonemapping.html [hunk 1]
```diff
@@ -162,6 +171,35 @@
...
+
+				environmentFolder.add( envParams, 'intensity', 0, 2, 0.1 )
+
+					.onChange( function ( value ) {
+
+						scene.environmentIntensity = value;
+						render();
+
+					} );
+
```
Comment (Mugen87) [inline examples/webgl_tonemapping.html:202]: Thanks for creating the PR but I would try to keep `webgl_tonemapping` as focused as possible. The rotation feature is already demonstrated in `webgl_materials_envmaps` and that should be sufficient. 

The addition of `environmentIntensity` is okay, imo.

Comment (WestLangley) [reply examples/webgl_tonemapping.html:202]: I agree with what @Mugen87 said.

Comment (vis-prime) [reply examples/webgl_tonemapping.html:202]: Oh , as long as it's available to test in one example it's good, will close this for now

**Comment by vis-prime** [comment]
the demo of these properties exists in other examples so closing this

**Pull request closed without merging**